src/components/compound/DlCard/DlCard.vue

```diff
@@ -51,9 +41,9 @@
                     :style="imageStyles"
                     :alt="imageAlt"
                     class="card--image__image-holder__image"
-                    :onload="onImageLoad"
```

Comment:
Need to check if we have places in the platform that use dl-card with onload as a prop and not as a function

Reply:
@guyDataloop 
In OA, we use the dl-card component twice, and in these instances, we did not use the onload.